CHANGELOG.md

```diff
@@ -6,6 +6,9 @@
 - Running `rails-response-dumper` with no arguments now limits discovery to
   `*_dumper.rb` files instead of all `*.rb` files.
 
+- Tweak tests on `stdout` so they don't count '.'s, but rather expect any nonzero
+  number of them, so they don't need to be updated with every new test dumper.
```

Comment:
The changelog only needs to mention user facing details. Testing is internal to the project, so can remove this.

```suggestion
```